### DISCUSSION TITLE
Chrome crashes when inserting large amount of data

### DISCUSSION THREAD

**Discussion by ManuFdezMontejano** [ujjwalguptaofficial/JsStore#268] [Q&A]
I'm using JsStore to create a database with 8 tables with 500.000 rows each.
When inserting that amount of data google chrome crashes completely, which does not happen when we reduce the number of rows and tables. We can see a strong use of the CPU when doing the insert. Also, we cannot use web workers in our environment.
What can be causing that crash? Any ideas of how to avoid it?
Thanks in advance.

**Comment by ujjwalguptaofficial** [comment]
storing that much of data and doing operation in client side is not recommended. it will be slow when runing select query.

For now - you can run multiple insert query by splitting your data into multiple parts.
